app/webauthn/webauthn_schema.py

```diff
@@ -0,0 +1,23 @@
+post_create_webauthn_credential_schema = {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "description": "POST webauthn_credential schema",
+    "type": "object",
+    "properties": {
+        "name": {"type": "string"},
+        "credential_data": {"type": "string"},
+        "registration_response": {"type": "string"},
+    },
+    "required": ["name", "credential_data", "registration_response"],
```

Comment:
Does this allow for blank names (empty string)? We decided we would allow this, since they're for the user's convenience (so it's not worth the effort doing validation).

Reply:
Update: I checked myself by modifying the tests, and it does allow a blank name. A very minor suggestion would be to have a test for this edge case.

Reply:
i'd rather not allow blank names. it will lead to more confusion than not. i'd rather update this to specifically check the length

Reply:
Hmm...I'm less certain, especially as it'll mean more work in the Admin app to handle / prevent the errors.

I think we can assume most people won't deliberately erase the name, and either enter their own, or stick with the default of "Unnamed key".

What's the downside of allowing a blank name?

Reply:
it's a massive usability/debugability issue as the string won't be truthy, and for example has been a big pain point/gross hack in our handling of placeholders where via the api you can send in empty string to have a blank placeholder but in the admin app you can't do that if sending a one-off message. i'm happy to put my foot down on this. 

if we really don't want to force people to write a name, even a placeholder name like "default key" or whatever, then we should make the string nullable but enforce that if it is not null it has length of at least 1.

Reply:
That's interesting to hear it's been an issue elsewhere. Empty strings aren't truthy in Ruby, so that one didn't occur to me. I don't _think_ it would be an issue, since the name is just something we show to the user (we don't do any processing on it). However, I also agree that there's an element of best practice here, so let's just require a min length.

I've made a note in the planning doc, so whoever does the update/remove card in Admin knows to support it.